src/pages/contributing/component/index.mdx

```diff
@@ -521,19 +521,219 @@ leaving any other comments on
 [GitHub](https://github.com/carbon-design-system/carbon-website/issues/new?assignees=&labels=feedback&template=feedback.md).
 ```
 
-### Style guidance
+### Style template
 
-We're still working on this template. We'd love your
-[help and feedback](https://github.com/carbon-design-system/carbon-website/issues/1246)!
+The style template helps describe how a component looks, including visual
+specifications such as color, typography, structure, and size.
 
 ```markdown
-Help wanted!
+---
+title: [Component name]
+description: [Explains what the component is in one or two sentences.]
+tabs: ['Usage', 'Style', 'Code', 'Accessibility']
+---
+
+<PageDescription>
+
+The following page documents visual specifications such as color, typography,
+structure, and size.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Color</AnchorLink> <AnchorLink>Interactive states</AnchorLink>
+<AnchorLink>Typography</AnchorLink> <AnchorLink>Structure</AnchorLink>
+<AnchorLink>Size</AnchorLink> <AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+## Color
+
+<!--
+Use a table to document properties and color tokens in this section. You can reference this [table generator](https://www.tablesgenerator.com/markdown_tables) for easy table creation.
+
+The following table shows an example from [Accordion](https://carbondesignsystem.com/components/accordion/style).
+-->
+
+| Element | Property      | Color token        |
+| ------- | ------------- | ------------------ |
+| Title   | color         | `$text-primary`    |
+| Content | color         | `$text-primary`    |
+| Icon    | fill          | `$icon-primary`    |
+| Header  | border-top    | `$border-subtle`\* |
+|         | background    | Transparent        |
+| Panel   | border-bottom | `$border-subtle`\* |
+|         | background    | Transparent        |
+
+<Caption>
+  * Denotes a contextual color token that will change values based on the layer
+  it is placed on.
+</Caption>
+
+<Row>
+<Column colLg={8}>
+
+![enabled state](images/accordion-style-1.png)
+
+</Column>
+</Row>
+
+## Interactive state color
+
+<!--
+Use a table to document properties and color tokens for interactive states. Begin the table with the State column, not the Element column; this helps users scan faster and better reference accompanying images, which are also organized by state. You can reference this [table generator](https://www.tablesgenerator.com/markdown_tables) for easy table creation.
+
+You may also choose to include images or tabs to illustrate variants.
+
+The following table shows an example from [Accordion](https://carbondesignsystem.com/components/accordion/style).
+-->
+
+| State    | Element | Property   | Color token      |
+| -------- | ------- | ---------- | ---------------- |
+| Hover    | Header  | background | `$layer-hover`   |
+| Focus    | Header  | border     | `$focus`         |
+| Disabled | Header  | border-top | `$border-subtle` |
+| Disabled | Title   | background | `$text-disabled` |
```

Comment:
We typically don't repeat the state Disabled in this column for each additional row item, it usually only remains in the first column. This is Dropdown for example:
<img width="841" alt="Screen Shot 2023-04-19 at 12 14 21 PM" src="https://user-images.githubusercontent.com/43969356/233136779-c4000831-78fc-4587-adf2-2f805815b436.png">